tests/testthat/test-checkers.R

```diff
@@ -0,0 +1,50 @@
+test_that("checking single logicals", {
+ library(rlang)
+
+ variable <- "pie"
+ expect_snapshot(brulee:::check_single_logical(variable), error = TRUE)
```

Comment:
```suggestion
 expect_snapshot(check_single_logical(variable), error = TRUE)
```

In general, don't need to namespace functions from the package being tested.

Reply:
That was left over from testing (since we can source these standalone files)